Sources/MapboxMaps/Foundation/MapboxMap.swift

```diff
@@ -502,6 +502,15 @@ extension MapboxMap {
     }
 }
 
+// MARK: - Attribution -
+
+extension MapboxMap: AttributionDataSource {
+    internal func attributions() -> [Attribution] {
+        let attributions = Attribution.parse(style.sourceAttributions())
+        return attributions
+    }
+}
+
 // MARK: - Testing only! -
 
```

Comment:
I know you did not edit this as part of this PR, but I think adding code just for testing in the class itself is not a good practice. Test code extensions can maybe sit in the test classes or something. It also raises an interesting question of why this is needed and if we can refactor. 

Although this is probably a separate conversation